index.html

```diff
@@ -0,0 +1,139 @@
+          </p>
+          <p>
+            But, the most widely accepted classification of musical instruments is the way they produce sound. The study of musical instruments is known as organology.
+          </p>
+          <p>
+            If you are interested in learning musical instruments, you must familiarize yourself with all of their types. Even if you are not interested in playing one, it is still exhilarating to explore different musical instruments.
+          </p>
+          <p>
+            It will help you to decide what kind of learning you wish to attain in the niche of music. So, here is the list of all the major classes of musical instruments.
+          </p>
+        </article>
+      </section>
+      <section class="main-section" id="wind_instruments">
+        <header>Wind Instruments</header>
+        <article>
+          <code></code>
+          <p>
+            In this type of musical instrument, all you need to do is blow into the musical instrument by following a defined order to produce a particular kind of sound. These instruments essentially work on the principles of sound waves, resonance, and harmonics. We can control the frequencies, and acoustics can be set and altered by changing how we blow into the instrument and what order we follow.
+          </p>
+          <p>
+            There is a wide variety of wind instruments that can be seen around the world. Some of the highly renowned wind instruments are:
+          </p>
+          <ul>
+            <li>Flute</li>
+            <li>Oboe</li>
+            <li>Clarinet</li>
+            <li>Bassoon</li>
+            <li>Saxophone</li>
+          </ul>
+        </article>
+      </section>
+      <section class="main-section" id="string_instruments">
+        <header>String Instruments</header>
+        <article>
+          <code></code>
+          <p>
+            The instruments that fall in this category consist of strings that can be vibrated to produce different sounds with varying pitches. The pitch of the sound produced via these instruments is highly dependent on the length of the air column inside the instrument. The pitch also significantly depends upon the thickness of the strings used in the instrument.
+          </p>
+          <p>
+            Some examples of string instruments include:
+          </p>
+          <ul>
+            <li>Guitar</li>
+            <li>Violin</li>
+            <li>Viola</li>
+            <li>Double Bass</li>
+            <li>Cello</li>
+            <li>Harp</li>
+          </ul>
+        </article>
+      </section>
+      <section class="main-section" id="percution_instruments">
+        <header>Percution Instruments</header>
+        <article>
+          <code></code>
```

Comment:
Could you please consider removing the empty tags

Reply:
- [ ] Here also, please check my comments on line `47` above on the `why` implementation 👍